deploy/helm/flightctl/charts/keycloak/templates/keycloak-realm-secret.yaml

```diff
@@ -1,27 +1,26 @@
-{{- $demoUserPassword := (include "keycloak.lookupOrGeneratePassword" (dict "secret" "keycloak-demouser-secret" "namespace" .Values.namespace "key" "password" "context" $)) }}
-{{- if .Values.realm.demoUser.enabled }}
+{{ if and (eq (.Values.global).target "standalone") (eq (.Values.global.auth).type "builtin") }}
+{{- $demoUserPassword := (include "keycloak.lookupOrGeneratePassword" (dict "secret" "keycloak-demouser-secret" "namespace" .Release.Namespace "key" "password" "context" $)) }}
 apiVersion: v1
 kind: Secret
 metadata:
   name: keycloak-demouser-secret
-  namespace: {{ .Values.namespace }}
+  namespace: {{ .Release.Namespace }}
 type: Opaque
 data:
-  username: {{ .Values.realm.demoUser.username | b64enc | quote }}
+  username: demouser
```

Comment:
being in the data section don't we still need to do b64enc?

Reply:
You're right. We need to b64 encode. Fixed